radis/lbl/base.py

```diff
@@ -2831,9 +2831,7 @@ def _cutoff_linestrength(self, cutoff=None):
 
         # Estimate time gained
         expected_broadening_time_gain = (
-            self._broadening_time_ruleofthumb
-            * Nlines_cutoff
-            * len(self.wbroad_centered)
+            self._broadening_time_ruleofthumb * Nlines_cutoff
```

Comment:
as you suggest in your PR, you can remove this entirely. To add again once we have better time estimate of how long a calculation takes based on the number of lines and the method used (that's your GSOC project!). Keep a TODO here maybe?